include/avif/avif.h

```diff
@@ -68,6 +68,10 @@ typedef int avifBool;
 
 #define AVIF_DIAGNOSTICS_ERROR_BUFFER_SIZE 256
 
+// A reasonable default for maximum image size to avoid out-of-memory errors or integer overflow in
+// (32-bit) int or unsigned int arithmetic operations.
+#define AVIF_DEFAULT_MAX_IMAGE_SIZE (16384 * 16384)
```

Comment:
Joe: Would you like me to rename this macro `AVIF_DEFAULT_IMAGE_SIZE_LIMIT` to match the naming convention of `AVIF_DEFAULT_IMAGE_COUNT_LIMIT`? (See line 76 below.)

Reply:
I don't have a strong preference here, but having them match might be nice.

Reply:
Done. `AVIF_DEFAULT_MAX_IMAGE_SIZE` has been renamed `AVIF_DEFAULT_IMAGE_SIZE_LIMIT`.